src/program/lwaftr/virt/lwaftrctl

```diff
@@ -49,16 +48,17 @@ function qemu_start_vm {
         "sudo numactl -m ${NUMA_NODE} taskset -c ${QEMU_CORE} \
             $QEMU $QEMU_ARGS \
```

Comment:
Correct, for NixOS we need `QEMU_ARGS` for passing initrd. I don't think it hurts if it's unset since bash uses `""` for unset variables (unless testing with `-z` is set).